File: mesh/android/ExoPlayerMeshJava/app/src/main/java/io/streamroot/ctl/delivery/client/mesh/exoplayermesh/Application.java
package io.streamroot.ctl.delivery.client.mesh.exoplayermesh;

import androidx.multidex.MultiDexApplication;

import io.streamroot.lumen.delivery.client.core.LumenDeliveryClient;

public class Application extends MultiDexApplication {
    @Override
    public void onCreate() {
        super.onCreate();
        LumenDeliveryClient.initializeApp(this);
    }
}

File: mesh/android/ExoPlayerMesh/app/src/main/java/io/streamroot/ctl/delivery/client/mesh/exoplayermesh/Application.kt
package io.streamroot.ctl.delivery.client.mesh.exoplayermesh

import androidx.multidex.MultiDexApplication
import io.streamroot.lumen.delivery.client.core.LumenDeliveryClient

class Application : MultiDexApplication() {
    override fun onCreate() {
        super.onCreate()
        LumenDeliveryClient.initializeApp(this)
    }
}